docs/getting-started/index.md

```diff
@@ -0,0 +1,250 @@
+---
+title: "Getting started with Apache Gravitino"
+slug: /getting-started
+license: "This software is licensed under the Apache License version 2."
+---
+
+There are several options for getting started with Apache Gravitino.
+
+<!--Docker option-->
+Installing and configuring Hive and Trino can be a little complex.
+If you are unfamiliar with the technologies, using Docker might be a good choice.
+There are pre-packaged containers for Gravitino, Apache Hive, Apache Hadoop,
+Trino, MySQL, PostgesSQL, and others,
+check [installing Gravitino playground](./playground.md) for more details.
+
+<!--Build from source-->
+This page guides you through the process to download and install Gravitino
+from source.
```

Comment:
Breaking this over two lines hides errors. This is probably better "This page guides you through the process of downloading and installing Gravitino from the source."

Reply:
Will fix.